tests/test_integrations/test_nessus.py

```
@@ -0,0 +1,49 @@
+        assert details["compliance_supported"] is False
+        assert "supported_interfaces" in details
+        assert "python_code_sample" in details
+        
+        # Verify the requested "preview" tag is present
+        assert details.get("preview") is True
+
+    def test_get_python_sdk_client(self):
+        creds = RestAPICreds(
+            url="https://test:8834",
+            access_key="my_access_key",
+            secret_key="my_secret_key"
```

Ruff (0.15.18)

[error] Possible hardcoded password assigned to argument: "secret_key"

(S106)